core-tests/shared/src/test/scala/zio/ZIOSpec.scala

```diff
@@ -1,24 +1,24 @@
 package zio
 
-class ZIOSpec extends BaseCrossPlatformSpec {
-  def is = "ZIOSpec".title ^ s2"""
-    ZIO.parallelErrors
-      Returns a list of 2 errors $allFailures
-      Returns a list of 1 error $oneFailure
+import zio.test._
+import zio.test.Assertion._
 
-  """
-
-  def allFailures =
-    for {
-      f1     <- IO.fail("error1").fork
-      f2     <- IO.fail("error2").fork
-      errors <- f1.zip(f2).join.parallelErrors[String].flip
-    } yield errors must_=== ::("error1", List("error2"))
-
-  def oneFailure =
-    for {
-      f1     <- IO.fail("error1").fork
-      f2     <- IO.succeed("success1").fork
-      errors <- f1.zip(f2).join.parallelErrors[String].flip
-    } yield errors must_=== ::("error1", List())
-}
+object ZIOSpec
+    extends ZIOBaseSpec(
+      suite("ZIOSpec")(
+        testM("oneFailure") {
+          for {
+            f1     <- IO.fail("error1").fork
+            f2     <- IO.succeed("success1").fork
+            errors <- f1.zip(f2).join.parallelErrors[String].flip
+          } yield assert(errors, equalTo(::("error1", List())))
```

Comment:
Nitpick: first time I see prefix `::` together with `List()`, could you, please, use something more standard: `"error1" :: Nil` or `List("error1")`?
I'm aware it was like it before.